iam_units/update.py

```diff
@@ -0,0 +1,64 @@
+from pathlib import Path
+
+import pandas as pd
+import sys
+
+
+DATA_PATH = Path(__file__).parent / 'data'
+
+# list of contexts
+contexts = ['SARGWP100', 'AR4GWP100', 'AR5GWP100']
```

Comment:
suggest to rename to `gwp_contexts` assuming that we'll want to add other context-types in the future (e.g., currency conversion)

Reply:
Will move it inside the function, should have done that in the first place.